run.py

```diff
@@ -4,14 +4,15 @@
 from controllers import db
 
 
-from models.prediction_model import Prediction
```

Comment:
It looks like this whole file was added to .gitignore, which makes sense.  So this should be removed from our git repo entirely to avoid future confusion.

Reply:
as i added the file into gitignore, have to manually delete from git repo